include/manifold/manifold.h

```diff
@@ -16,11 +16,6 @@
 #include <functional>
 #include <memory>  // needed for shared_ptr
 
-#if defined(MANIFOLD_EXPORT) || defined(MANIFOLD_DEBUG)
-#include <iostream>
-#include <string>
-#endif
-
```

Comment:
Just a thought - should we be hiding the ReadOBJ/WriteOBJ methods behind an #ifdef MANIFOLD_DEBUG as well, since they use std::istream and std::ostream?

Reply:
Actually, nevermind - I'll fix that myself in a separate PR - should be orthogonal to this.